hw/ip/otbn/pre_syn/syn_setup.example.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+# Setup IP name and top module.
+# When changing the top module, some parameters might not exist and
+# tcl/yosys_run_synth.tcl might need to be adjusted accordingly.
+# Known working top modules are: otbn_core, otbn
+export LR_SYNTH_IP_NAME=otbn
+export LR_SYNTH_TOP_MODULE=otbn_core
```

Comment:
@abdullahvarici , please use `otbn_core` for the time being. Similar to the Ibex example where `ibex_top` is used which also doesn't contain the memories, you should be able to embed the synthesized netlist inside the top file together with the unsynthesized memories for the parse and trace steps.